src/presenters/admin/migration-error-presenter.php

```diff
@@ -61,7 +66,8 @@ public function present() {
 		);
 
 		return \sprintf(
-			'<div class="notice notice-error"><p>%1$s</p><p>%2$s</p><p>%3$s</p>%4$s</div>',
+			'<div class="notice notice-error yoast-migrated-notice"><h4 class="yoast-notice-migrated-header">%1$s</h4><div class="notice-yoast-content"><p>%2$s</p><p>%3$s</p><p>%4$s</p>%5$s</div></div>',
```

Comment:
This is an `h4`, and the Free ones are `h2` which causes a difference in styling. I would recommend adding some additional styling to the `yoast-notice-migrated-header` class to make up for it.

Actually, `h2` might even be more semantically correct? 🤔 

Whatever you add here, please take a look at the add-ons too! As they follow the same principle.

Reply:
I unified the styling via the `.yoast-notice-migrated-header.` CSS rule